### ISSUE TITLE
`@Suppress` doesn't work with `StringTemplate` rule

### ISSUE THREAD

**Issue by BraisGabin** [detekt/detekt#5053]
## Expected Behavior
If I annotate a class with `@Suppress("StringTemplate")` the rule `StringTemplate` doesn't raise any issue in that class.

## Observed Behavior
It does: https://github.com/detekt/detekt/runs/7255299587?check_suite_focus=true#step:4:540

## Steps to Reproduce
```kotlin
@Suppress("StringTemplate")
class A {
 val a = "a"
 val b = "${a}"
}
```

## Context
#5051

It works if I use `@file:Suppress("StringTemplate")`

## Your Environment
* Version of detekt used: main/1.21.0-RC2
* Gradle scan link: https://ge.detekt.dev/s/wzs6gfrjvtu2c

**Comment by BraisGabin** [comment]
Closing. I just remember that this is a known wont-fix
